src/app.css

```
@@ -1,47 +1,64 @@
   outline-offset: 2px;
 }
+
+/* ── Screen-reader only ── */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)